### ISSUE TITLE
Rolling back a release not working cause of "Undefined variable"

### ISSUE THREAD

**Issue by elfantastico** [andres-montanez/Magallanes#215]
I'm trying to rollback a release with 
 - mage releases rollback --release=0 to:vm
 - mage rollback -1 to:vm

and I'm getting a

PHP Notice:  Undefined variable: currentCopy in /opt/magallanes/1.0.5/Mage/Task/BuiltIn/Releases/RollbackTask.php on line 140
PHP Stack trace:
PHP   1. {main}() /opt/magallanes/1.0.5/bin/mage:0
PHP   2. Mage\Console->run() /opt/magallanes/1.0.5/bin/mage:34
PHP   3. Mage\Command\BuiltIn\RollbackCommand->run() /opt/magallanes/1.0.5/Mage/Console.php:146
PHP   4. Mage\Task\BuiltIn\Releases\RollbackTask->run() /opt/magallanes/1.0.5/Mage/Command/BuiltIn/RollbackCommand.php:69

Notice: Undefined variable: currentCopy in /opt/magallanes/1.0.5/Mage/Task/BuiltIn/Releases/RollbackTask.php on line 140

Call Stack:
    0.0003     230736   1. {main}() /opt/magallanes/1.0.5/bin/mage:0
    0.0013     310088   2. Mage\Console->run() /opt/magallanes/1.0.5/bin/mage:34
    0.0525     734880   3. Mage\Command\BuiltIn\RollbackCommand->run() /opt/magallanes/1.0.5/Mage/Console.php:146
    0.0540     859000   4. Mage\Task\BuiltIn\Releases\RollbackTask->run() /opt/magallanes/1.0.5/Mage/Command/BuiltIn/RollbackCommand.php:69

**Comment by bunin** [comment]
similar issue

running ``./vendor/bin/mage releases rollback --release=-1 to:production``

```
Running Rollback Release [id=20150410123533] ... PHP Notice:  Undefined variable: currentCopy in /vendor/andres-montanez/magallanes/Mage/Task/BuiltIn/Releases/RollbackTask.php on line 140
PHP Stack trace:
PHP   1. {main}() /vendor/andres-montanez/magallanes/bin/mage:0
PHP   2. Mage\Console->run() /vendor/andres-montanez/magallanes/bin/mage:34
PHP   3. Mage\Command\BuiltIn\ReleasesCommand->run() /vendor/andres-montanez/magallanes/Mage/Console.php:146
PHP   4. Mage\Task\BuiltIn\Releases\RollbackTask->run() /vendor/andres-montanez/magallanes/Mage/Command/BuiltIn/ReleasesCommand.php:85
FAIL
```

from log file:
```
2015-04-10 13:03:15 -- Running Rollback Release [id=20150410123533] ... 
2015-04-10 13:03:15 -- Run remote command cd /var/www && ls -ld releases/20150410123533 | awk '{print \$3\":\"\$4}'
2015-04-10 13:03:15 -- ---------------------------------
2015-04-10 13:03:15 -- ---- Executing: $ ssh  -p 22 -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no user@host "sh -c \"cd /var/www && ls -ld releases/20150410123533 | awk '{print \$3\":\"\$4}'\""
2015-04-10 13:03:15 -- awk: line 1: syntax error at or near :
2015-04-10 13:03:15 -- ---------------------------------
2015-04-10 13:03:15 -- Run remote command cd /var/www && ln -sfn  releases/20150410123533.tmp && mv -T releases/20150410123533.tmp current
2015-04-10 13:03:15 -- ---------------------------------
2015-04-10 13:03:15 -- ---- Executing: $ ssh  -p 22 -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no user@host "sh -c \"cd /var/www && ln -sfn  releases/20150410123533.tmp && mv -T releases/20150410123533.tmp current\""
2015-04-10 13:03:16 -- mv: cannot stat `releases/20150410123533.tmp': No such file or directory
2015-04-10 13:03:16 -- ---------------------------------
2015-04-10 13:03:16 -- FAIL
```

**Comment by eps90** [comment]
@elfantastico @bunin Hey, the issue has been resolved in #209 and the fix is still on `develop` branch. Try to pass `dev-develop` in `composer.json` as target version. I hope this fix gonna be released to stable version soon :)

**Comment by andres-montanez** [comment]
Hi! As pointed by @eps90 the new release fixes this!
Thanks!
